backup/setup/roles.json

```diff
@@ -64,7 +64,8 @@
 		"TOOL_NEW_VIEW",
 		"COURSE_EDIT",
 		"STUDENT_CREATE",
-		"TEACHER_CREATE"
```

Comment:
we have school scoped news right now only, right? Maybe this could be School_NEws_edit then if we add global or course news later